File: cmd/gui/vue/lib/html/overview.go
package html

import "github.com/p9c/pod/cmd/gui/mod"

func screenOverview() mod.DuOScomp {
	return mod.DuOScomp{
		IsApp:    false,
		Name:     "Screen Overview",
		ID:       "screenOverview",
		Version:  "0.0.1",
		CompType: "screen",
		SubType:  "screen",
		Js: `
	data () { return {
		duOSys,
}},
 methods: { 
  } 
`,
		Template: `<div id="screenoverview" class="Overview"><div id="panelwalletstatus" class="Balance"></div><div id="panelsend" class="Send"></div><div id="panelnetworkhashrate" class="NetHash"></div><div id="panellocalhashrate" class="LocalHash"></div><div id="panelstatus" class="Status"></div><div id="paneltxsex" class="Txs"></div><div class="Log"></div><div class="Info"></div><div class="Time"></div></div>`,
	}
}

File: pkg/gui/assets/w/js/pages/overview.js
var PageOverview = {
	el: "#overview",
  	name: "Overview",
  	template: `<main class="pageOverview">
	  <div id="panelwalletstatus" class="Balance">
		  <PanelBalance :balance="duoSystem.status.balance" :txsnumber="duoSystem.status.txsnumber"/>
	  </div>
	  <div id="panelsend" class="Send">
		  <PanelSend />
	  </div>
	  <div id="panelnetworkhashrate" class="NetHash">
	<PanelNetworkHashrate :hashrate="duoSystem.status.networkhashrate"/>
	  </div>
	  <div id="panellocalhashrate" class="LocalHash">
	  <PanelLocalHashrate :hashrate="duoSystem.status.hashrate"/> 
	  </div>
	  <div id="panelstatus" class="Status">
		  <PanelStatus :status="duoSystem.status"/>
	  </div>
	  <div id="paneltxsex" class="Txs">
	  <PanelLatestTx :transactions="duoSystem.transactions"/>
	  </div>
	  <div class="Log">
	  </div>
	  <div class="Info">
	  </div>
	  <div class="Time">
	  </div>
  </main>`,
  	components: {
		PanelBalance,
		PanelSend,
		PanelLatestTx,
		PanelLocalHashrate,
		PanelNetworkHashrate,
		PanelStatus
	}
}